extensions/applicationinsights-debugplugin-js/sample/src/index.js

```diff
@@ -0,0 +1,24 @@
+const polka = require("polka");
```

Comment:
I don't think we actually need this at all as you *should* be able to just load the index.html directly.
The only difference would be around cors related headers handling (which I recently auto disabled when the protocol is file://)